lib/SILOptimizer/Mandatory/TransferNonSendable.cpp

```diff
@@ -2162,14 +2138,26 @@ class PartitionAnalysis {
             auto expr = getExprForPartitionOp(partitionOp);
             function->getASTContext().Diags.diagnose(
                 expr->getLoc(), diag::arg_region_transferred);
-          });
+          };
+      eval.nonTransferrableElements = translator.getNeverTransferredValues();
+      eval.isActorDerivedCallback = [&](Element element) -> bool {
```

Comment:
I wonder if these callbacks are worth it at all, do they allow unit testing or something?

Reply:
I don't think they are. The idea is to make it so that one can visit/process the blocks and then reuse the same visiting code to visit/emit the errors. I'll see if I can come up with something better.